source/patterns/@aws-solutions-constructs/core/lib/websocket-api-defaults.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+export const DEFAULT_ROUTE_QUEUE_VTL_CONFIG =
```

Comment:
I understand what this is doing as I've studied it before. Is there anything we can do to generate some human-readable comments to help others?

Reply:
Split into smaller strings and concatenate them.